code/game/gamemodes/revolution/revolution.dm

```diff
@@ -14,6 +14,7 @@
 	antag_flag = ROLE_REV
 	false_report_weight = 10
 	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
+	required_jobs = list(list("Captain"=1),list("Head of Personnel"=1),list("Head of Security"=1),list("Chief Engineer"=1),list("Research Director"=1),list("Chief Medical Officer"=1)) //Any head present
```

Comment:
These strings aren't `#define`d anywhere?

Reply:
Nope, that's why i added typo check. Honestly most job checks should use define id's or straight checking types but consistency of these is a mess.